ferrilata-devilline.IntegrationTests/Scenarios/IntegrationTests.cs

```diff
@@ -12,5 +21,185 @@ public IntegrationTests(TestContext testContext)
+
+            var client = _testContext.Client;
+
+            var request = new HttpRequestMessage(HttpMethod.Post, url);
+
+            request.Content = new StringContent(PostingJson,
+                                    Encoding.UTF8,
+                                    "application/json");
+            request.Headers.Add("Authorization", "something");
+
+            var response = await client.SendAsync(request);
+
+            Assert.Equal(System.Net.HttpStatusCode.NotFound, response.StatusCode);
+        }
+
+        [Theory]
+        [InlineData("api/post/pitch")]
+        public async Task PostPitchOKPropertytAuthorizationOK_TestMessage(string url)
+        {
+            var newPosting = new Pitch
+            {
+                BadgeName = "English speaker",
+                OldLVL = 2,
+                PitchedLVL = 3,
+                PitchMessage = "Hello World! My English is bloody gorgeous.",
+                Holders = new[] { "balazs.jozsef", "benedek.vamosi", "balazs.barna" }.ToList()
+            };
+
+            var PostingJson = JsonConvert.SerializeObject(newPosting);
+
+            var client = _testContext.Client;
+
+            var request = new HttpRequestMessage(HttpMethod.Post, url);
+
+            request.Content = new StringContent(PostingJson,
+                                    Encoding.UTF8,
+                                    "application/json");
+            request.Headers.Add("Authorization", "something");
+
+            var response = await client.SendAsync(request);
+
+            string ResponseBody = await response.Content.ReadAsStringAsync();
+
+            Assert.Equal("Created",
+                JsonConvert.DeserializeObject<Dictionary<string, string>>(ResponseBody)["message"]);
+        }
+
+        [Theory]
+        [InlineData("api/post/pitch")]
+        public async Task PostPitchOKPropertytAuthorizationOK_TestUnauthorized(string url)
+        {
+            var newPosting = new Pitch
+            {
+                BadgeName = "English speaker",
+                OldLVL = 2,
+                PitchedLVL = 3,
+                PitchMessage = "Hello World! My English is bloody gorgeous.",
+                Holders = new[] { "balazs.jozsef", "benedek.vamosi", "balazs.barna" }.ToList()
+            };
+
+            var PostingJson = JsonConvert.SerializeObject(newPosting);
+
+            var client = _testContext.Client;
+
+            var request = new HttpRequestMessage(HttpMethod.Post, url);
+
+            request.Content = new StringContent(PostingJson,
+                                    Encoding.UTF8,
+                                    "application/json");
+
+            var response = await client.SendAsync(request);
+
+            string ResponseBody = await response.Content.ReadAsStringAsync();
+
+            Assert.Equal("Unauthorized",
+                JsonConvert.DeserializeObject<Dictionary<string, string>>(ResponseBody)["message"]);
```

Comment:
Read the Content of the body and assert it with the expected json string

Reply:
done